cli/lib/cert-lib.js

```diff
@@ -3,13 +3,13 @@ const request = require('request');
 const util = require('util');
 const fs = require('fs');
 const path = require('path');
-const yaml = require('js-yaml');
-const prompt = require('cli-prompt');
 const url = require('url');
 const pem = require('pem');
 const crypto = require('crypto');
 const async = require('async');
 const debug = require('debug')('cert')
+//const yaml = require('js-yaml');
```

Comment:
There are commented out lines of code in this and other files throughout this PR that look like dead code removal. We should probably just the remove code we're not using instead of leaving dead comments.